econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -104,60 +105,67 @@ def _get_metadata_causal_insights_keys():
             _CausalInsightsConstants.ViewKey]
 
 
-def _first_stage_reg(X, y, *, automl=True):
+def _first_stage_reg(X, y, *, automl=True, random_state=None, verbose=0):
     if automl:
-        model = GridSearchCVList([make_pipeline(StandardScaler(), LassoCV()),
+        model = GridSearchCVList([make_pipeline(StandardScaler(), LassoCV(random_state=random_state)),
                                   RandomForestRegressor(
-                                      n_estimators=100, random_state=123, min_samples_leaf=10),
-                                  lgb.LGBMRegressor(num_leaves=32)],
+                                      n_estimators=100, random_state=random_state, min_samples_leaf=10),
+                                  lgb.LGBMRegressor(num_leaves=32, random_state=random_state)],
                                  param_grid_list=[{},
                                                   {'min_weight_fraction_leaf':
                                                       [.001, .01, .1]},
                                                   {'learning_rate': [0.1, 0.3], 'max_depth': [3, 5]}],
-                                 cv=2,
-                                 scoring='neg_mean_squared_error')
+                                 cv=3,
+                                 scoring='r2_score',
```

Comment:
This should be 'r2', 'r2_score' throws an error
https://scikit-learn.org/stable/modules/model_evaluation.html

Reply:
Fixed, thanks